install/helm/agones/templates/controller.yaml

```diff
@@ -31,13 +32,13 @@ spec:
       app: {{ template "agones.name" . }}
       release: {{ .Release.Name }}
       heritage: {{ .Release.Service }}
-  replicas: 1
+  replicas: {{ $controllerReplica }}
   strategy:
     type: Recreate
   template:
     metadata:
       annotations:
-{{- if not $featureGates.SplitControllerAndExtensions }}
+{{- if not (gt (int $controllerReplica) 1) }}
```

Comment:
do you need the cast? or is it just protecting the case where they passed a non-int for `replicas`? If the latter, can we validate it above rather than casting throughout?

Reply:
there was a typing error when I was using the helper, shouldn't be an issue now.